src/qcasutreeitem.cpp

```diff
@@ -88,11 +94,9 @@ void QCasuTreeItem::resetSelection(){
 }
 
 void QCasuTreeItem::updateSelection(){
-
     child_selected = false;
     for(int k=0;k<this->childCount() - 1;k++) // LAST CHILD ARE SETPOINTS, NO NEED TO CHECK SELECTION
         for(int i=0;i<this->child(k)->childCount();i++) if(this->child(k)->child(i)->isSelected()) child_selected = true;
-
```

Comment:
Here was idea if groupTree item was selected, it would select group members treeItem accordingly but it is not possible for deselection would be painful process, and in every check it wouldn't know if it was deselected now or didn't change state (and that would raise question which group members treeItems to deselect and which not)

Workaround for this is simple but now pretty in QTrendPlot class.